### ISSUE TITLE
Inconsistency in import completion

### ISSUE THREAD

**Issue by vincentwoo** [ervandew/eclim#458]
If you use eclim to complete a statement like

```java
import java.
```

Eclim will helpfully return a bunch of results like

```
java.applet.*;
java.awt.*;
java.awt.color.*;
java.awt.datatransfer.*;
java.awt.dnd.*;
java.awt.dnd.peer.*;
java.awt.event.*;
java.awt.font.*;
```

However, this differs from the usual behavior because it includes the entire import path, starting with `java`. Because we lack a completion start column (as referenced in https://github.com/ervandew/eclim/issues/457), naive editor behavior here will be to produce a result like:

```java
import java.java.awt.color.*; // note the doubled java.java.
```

Would it be possible to either

a) Fix the related issue and return a start column or
b) return import results like how other completion results are normally returned (showing only continuations from the current token onwards):

```
applet.*;
awt.*;
awt.color.*;
awt.datatransfer.*;
awt.dnd.*;
awt.dnd.peer.*;
awt.event.*;
awt.font.*;
```

**Comment by ervandew** [comment]
That inconsistency on import completions is an inconsistency with eclipse. Eclim does very little modifications to the returned completion currently. It looks like I decided to handle that inconsistency on the vimscript side by removing the portion of the completion result that is already present in the file. This could be done on the eclimd side instead by reading the relevant line from the file performing the same essential logic currently found in eclim's java vim completion function.

**Comment by vincentwoo** [comment]
Resolved more generally by #459
